doc/user_guide/tools.rst

```diff
@@ -123,12 +123,12 @@ array of scalars by changing the default *view* of
 .. code-block:: python
 
     >>> scalar = hs.signals.BaseSignal(np.random.random((10, 20, 30)))
```

Comment:
Maybe just `scalar = hs.signals.BaseSignal(np.random.random((10, 20, 30))).T` would be better with a little explanation below or above?